Weblate: Update translations

photoprismci opened the pull request.

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Translation: PhotoPrism/Backend
Translate-URL: https://translate.photoprism.app/projects/photoprism/backend/

weblate committed:
Translated using Weblate (Hebrew)

Currently translated at 97.2% (71 of 73 strings)

Translation: PhotoPrism/Backend
Translate-URL: https://translate.photoprism.app/projects/photoprism/backend/he/

weblate committed:
Translated using Weblate (Czech)

Currently translated at 100.0% (73 of 73 strings)

Translation: PhotoPrism/Backend
Translate-URL: https://translate.photoprism.app/projects/photoprism/backend/cs/

weblate committed:
Merge remote-tracking branch 'origin/develop' into develop

weblate committed:
Merge remote-tracking branch 'origin/develop' into develop

weblate committed:
Merge remote-tracking branch 'origin/develop' into develop

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/photoprism/photoprism?pullRequest=2137) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/photoprism/photoprism?pullRequest=2137) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/photoprism/photoprism?pullRequest=2137) it.</sub>

weblate committed:
Merge remote-tracking branch 'origin/develop' into develop

weblate committed:
Merge remote-tracking branch 'origin/develop' into develop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: PhotoPrism/Backend
Translate-URL: https://translate.photoprism.app/projects/photoprism/backend/
